docs/framing.md

```diff
@@ -2,6 +2,46 @@
 
 Framing wraps messages with headers and checksums so receivers can identify message boundaries and verify integrity.
 
+## Frame Format Definitions
+
+All supported frame formats are defined in [`examples/frame_formats.proto`](../examples/frame_formats.proto). This file provides:
+
+- Protocol Buffer definitions for each frame format
+- Enumeration of all frame format types
+- Configuration message for runtime format selection
+
+## Recommended Frame Formats
+
+| Format | Preamble | Header | CRC | Length | Total Overhead | Use Case |
```

Comment:
Lets remove the header field completely. So we just have preamble, length field, CRC and then total overhead.

The length field should not be 8-bit or 16-bit but just 1,2. It should be clear that this table is showcasing number of bytes.